docs/johannes_mueller/yolo_from_omero/train_yolo.ipynb

```diff
@@ -0,0 +1,705 @@
+    "\n",
+    "<img src=\"./imgs/labels.jpg\" alt=\"labels\" style=\"width:50%;\">"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Evaluation\n",
+    "\n",
+    "Once the training is done, we should have a look at some of the metrics yolo generates for us. For instance, we are shown some predictions and the corresponding ground truth labels. This is a good way to see how well the model is doing.\n",
+    "\n",
+    "| Labels | Predictions |\n",
+    "|--------|-------------|\n",
+    "| <img src=\"./imgs/val_batch1_labels.jpg\" alt=\"labels\" style=\"width:100%;\"> | <img src=\"./imgs/val_batch1_pred.jpg\" alt=\"predictions\" style=\"width:100%;\"> |\n",
+    "\n",
+    "At a glance - not half-bad, right? Besides just looking at the predicted bounding boxes we can - and we also should - look at some metrics. What's very important here, is the confusion matrix produced by the training, which essentially tells us how well the model is doing in terms of precision and recall.\n",
+    "\n",
+    "| Counts | Normalized |\n",
+    "|--------|------------|\n",
+    "| <img src=\"./imgs/confusion_matrix.png\" alt=\"confusion matrix\" style=\"width:100%;\"> | <img src=\"./imgs/confusion_matrix_normalized.png\" alt=\"confusion matrix normalized\" style=\"width:100%;\"> |\n",
+    "\n",
+    "From this we learn that our model is rather mediocre at detecting the `cell` class: In the normalized confusion matrix, we see that the True positive rate is about 50% for the cell class, whereas the model often (42%) predicts background (no object) where we annotated a cell. Interestingly, the model often predicts cells where the annotator only saw background.\n",
+    "\n",
+    "For the spheroids, we are quite good at finding them, with a true positive rate of 84%. There is one way, of course to improve performance: \n",
+    "\n",
+    "*Annotate more data!*\n",
+    "\n",
+    "But that's a story for another day. For now, we have a working model that can detect cells and spheroids in our images.\n",
+    "\n",
+    "## Applying the model\n",
+    "\n",
+    "Lastly, if you want to apply the model on some new data, this can be done fairly easily with YOLO. In a real applciation, replace `list_of_other_images` with a list of paths to your images. Yolo automatically saves two model checkpoints for us to use: The best model and the last model. Obviously, we'll want to use the best model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = YOLO(\"./runs/detect/train/weights/best.pt\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "list_of_other_images = [os.path.join(output_dirs['test']['images'], f) for f in test_files]\n",
```

Comment:
This part is a bit difficult to understand. It would be better, if the example was a bit easier to understand, since this would be something the users would want to modify.